src/app/qgisappstylesheet.cpp

```diff
@@ -152,6 +152,23 @@ void QgisAppStyleSheet::buildStyleSheet( const QMap<QString, QVariant> &opts )
                                     "    padding-right: 0px;"
                                     "}" ).arg( frameMargin );
 
+    style += QStringLiteral( "QTreeView#mOptionsTreeView {"
+                             "    background-color: rgba(69, 69, 69, 0);"
+                             "    outline: 0;"
+                             "}"
+                             "QFrame#mOptionsListFrame {"
+                             "    background-color: rgba(69, 69, 69, 220);"
+                             "}"
+                             "QTreeView#mOptionsTreeView::item {"
+                             "    color: white;"
+                             "    padding: %1px;"
+                             "}"
+                             "QTreeView#mOptionsTreeView::item::selected {"
```

Comment:
```suggestion
                             "QTreeView#mOptionsTreeView::item::selected, QTreeView#mOptionsTreeView::branch::selected {"
```

![image](https://user-images.githubusercontent.com/1728657/126926541-b2b85565-4627-4c9e-adb6-c479686a5b94.png)

Reply:
(@nyalldawson , ping)

Reply:
thank you!